vignettes/ArrayExpress-example.Rmd

```diff
@@ -223,10 +222,20 @@ We associate active signatures to known KEGG pathways.
 
 ```{r}
 pathway_result <- annotate_signatures_with_genesets(
-  selected_signatures = active_sigs, model = model, genesets = KEGG)
+  selected_signatures = unique_active_sigs, model = model, genesets = KEGG)
 DT::datatable(pathway_result)
 ```
 
+In this case, there are also signatures uncharacterized by KEGG.
+```{r}
+uncharacterized_sigs <- setdiff(unique_active_sigs, pathway_result$signature)
+uncharacterized_sigs
+```
+
+Check the signature similarity of the uncharacterized signatures.
+```{r}
+plot_signature_overlap(selected_signatures = uncharacterized_sigs)
```

Comment:
In line162, I saved it to a variable so that I can retrieve the signature order from the saved variable for other uses. Here I only want to generate a plot and do not need to use the order later. It will make a plot no matter it is saved to a variable or not.